hca/dss/__init__.py

```diff
@@ -100,89 +102,184 @@ def download(self, bundle_uuid, replica, version="", dest_name="",
             if not os.path.isdir(dest_name):
                 os.makedirs(dest_name)
 
-            logger.info("File %s: Retrieving...", filename)
             file_path = os.path.join(dest_name, filename)
+            logger.info("File %s: Retrieving...", file_uuid)
+            self._download_file(file_uuid, file_["sha256"], file_path, replica, file_['size'],
+                                file_version=file_version, num_retries=num_retries, min_delay_seconds=min_delay_seconds)
+
+    def _download_file(self,
+                       file_uuid,
+                       file_sha256,
+                       dest_path,
```

Comment:
Nit: `dest_path` shouldn't be renamed to `file_path` as it's a file and not a destination.